libqtile/command/base.py

```diff
@@ -128,7 +128,7 @@ def _select(self, name: str, sel: Optional[Union[str, int]]) -> Optional[Command
         Return None if no such object exists
         """
 
-    def command(self, name: str) -> Callable:
+    def command(self, name: str) -> Optional[Callable]:
```

Comment:
The alternative to returning None would be to define a `no_op` command and return that.

Reply:
I think `None` is the right call because it *is* right.